cli/tink/cmd/workflow/create.go

```diff
@@ -40,14 +35,14 @@ var createCmd = &cobra.Command{
 func addFlags() {
 	flags := createCmd.PersistentFlags()
 	flags.StringVarP(&template, "template", "t", "", "workflow template")
-	flags.StringVarP(&target, "target", "r", "", "workflow target")
+	flags.StringVarP(&hardware, "hardware", "r", "", "workflow target hardwares")
```

Comment:
Change the description from "workflow target hardwares" to "workflow hardware" or "workflow targeted hardware".

Reply:
I think `workflow targeted hardware` suits better so channged.